platforms/karaf/features/src/main/resources/features.xml

```diff
@@ -1397,6 +1397,7 @@
     <bundle>mvn:org.apache.camel/camel-linkedin/${project.version}</bundle>
   </feature>
   <feature name='camel-leveldb' version='${project.version}' resolver='(obr)' start-level='50'>
+    <feature prerequisite="true">wrap</feature>
```

Comment:
@gnodet I have not seen this attribute before ` prerequisite` in use in karaf features. Is this a new thing or in other words does it work on karaf 4.0.x onwards or 4.1.x onwards. And what does it really do?